cypress/fixtures/holderChange.json

```diff
@@ -17,5 +16,8 @@
   "language": "ca_ES",
   "iban": "ES77 1234 1234 1612 3456 7890",
   "voluntaryCent": true,
-  "becomeMember": true
+  "becomeMember": true,
+  "noBecomeMember": false,
```

Comment:
It is weird having two booleans with inverse logic meanings but having not always complementary values. I would rename `noBecomeMember` to `holderChangeTrialPeriod` or something more explicit on the usecase we are dealing with here. Maybe next month we have another exception for people contracting with American Express or coming from Candela.

Reply:
We think that when those cases arrive, then we will change that naming if necessary. Now, even though the two boolean with inverse logic might seem strange, we think that it is easy to understand (as the main case).